omero/developers/index.rst

```diff
@@ -31,7 +31,7 @@ with commercial licenses and customization available from |Glencoe|. You can
 read about how OMERO has developed since the project started in the
 :doc:`/users/history`.
 
-.. _GNU General Public License (GPL): https://www.gnu.org/copyleft/gpl.html
+.. _GNU General Public License (GPL): https://www.gnu.org/licenses/gpl-3.0.html
```

Comment:
An alternative would be
```suggestion
.. _GNU General Public License (GPL): https://www.gnu.org/licenses/licenses.html#GPL
```

As this link is specific to GPL 3 while some of the code is under GPL 2

Reply:
This will work.
We did not see the error again on latest-ci